packages/3dt/build

```diff
@@ -36,7 +36,7 @@ EOF
 
 cat <<EOF | tee $master_service
 [Unit]
-Description=Diagnostics: DC/OS Distributed Diagnostics Tool Master API and Aggregation Service
+Description=DC/OS Diagnostics Master: diagnostics aggregation service
```

Comment:
Doesn't this also collect diagnostics on the master? so it's not just for aggregating from the agents.